docs/getting_started/hopsworksai/guides/eks_ecr_integration.rst

```diff
@@ -0,0 +1,319 @@
+          - system:nodes
+          rolearn: arn:aws:iam::755182613526:role/eksctl-my-eks-cluster-nodegroup-m-NodeInstanceRole-FQ7L0HQI4NCC
+          username: system:node:{{EC2PrivateDNSName}}
+    kind: ConfigMap
+    metadata:
+    creationTimestamp: "2020-08-24T07:42:31Z"
+    name: aws-auth
+    namespace: kube-system
+    resourceVersion: "770"
+    selfLink: /api/v1/namespaces/kube-system/configmaps/aws-auth
+    uid: c794b2d8-9f10-443d-9072-c65d0f2eb552
+
+Add another item to *mapRoles* to add *system:masters* priviliges to your instance profile, check lines 13-16 in the following output. Make sure to replace 'YOUR INSTANCE PROFILE ARN' with your instance profile arn before saving.
+
+.. code-block:: bash 
+    :number-lines:
+
+    # Please edit the object below. Lines beginning with a '#' will be ignored,
+    # and an empty file will abort the edit. If an error occurs while saving this file will be
+    # reopened with the relevant failures.
+    #
+    apiVersion: v1
+    data:
+    mapRoles: |
+        - groups:
+          - system:bootstrappers
+          - system:nodes
+          rolearn: arn:aws:iam::755182613526:role/eksctl-my-eks-cluster-nodegroup-m-NodeInstanceRole-FQ7L0HQI4NCC
+          username: system:node:{{EC2PrivateDNSName}}
+        - groups:
+          - system:masters
+          rolearn: <YOUR INSTANCE PROFILE ARN>
+          username: hopsworks
+    kind: ConfigMap
+    metadata:
+    creationTimestamp: "2020-08-24T07:42:31Z"
+    name: aws-auth
+    namespace: kube-system
+    resourceVersion: "770"
+    selfLink: /api/v1/namespaces/kube-system/configmaps/aws-auth
+    uid: c794b2d8-9f10-443d-9072-c65d0f2eb552
+
+Once you are done with editing the configmap, click ESC-X to save the updated config map.
+
+.. code-block:: bash
+
+    configmap/aws-auth edited
+
+Step 4: Open Hopsworks required ports on your EKS cluster security group 
+-----------------------------------------------------------------------
+You need to open Http (80) and Https (443) ports on the security group of your eks cluster.
+First, you need to get the name of the security group of your eks cluster by using the following eksctl command.
+
+.. code-block:: bash
+    
+    eksctl utils describe-stacks --region=us-east-2 --cluster=my-eks-cluster
+
+Check the output for *eksctl-my-eks-cluster-cluster::ClusterSecurityGroupId* 
+
+.. code-block:: bash
+
+    ...
+    {
+      ExportName: "eksctl-my-eks-cluster-cluster::ClusterSecurityGroupId",
+      OutputKey: "ClusterSecurityGroupId",
+      OutputValue: "YOUR_EKS_SECURITY_GROUP_ID"
+    },
+    ...
+
+Once you get the security group id (YOUR_EKS_SECURITY_GROUP_ID), you need to proceed to the AWS management console to open the required ports by clicking on the following link (make sure to replace YOUR_EKS_SECURITY_GROUP_ID with the security group id):
+
+https://us-east-2.console.aws.amazon.com/ec2/v2/home?region=us-east-2#ModifyInboundSecurityGroupRules:securityGroupId=YOUR_EKS_SECURITY_GROUP_ID
```

Comment:
you can't click on the link if you need to modify it. I would point to `https://us-east-2.console.aws.amazon.com/ec2/v2/home?region=us-east-2#SecurityGroups:` and then instruct the user to select the security group and to click on `edit inbound rules`